packages/api/src/trpc.ts

```diff
@@ -62,13 +64,27 @@ export const loggedProcedure = t.procedure.use(async (opts) => {
   const session = opts.ctx.session;
   const email = session?.user.email ?? "unknown";
   const source = opts.ctx.trpcSource;
+  const error =
+    result.ok === false
```

Comment:
if result.ok is false, then there's always an error object in the result, so just appending that to the meta to get more information in the warn statemnet

Reply:
otherwise some stuff was being swallowed like auth checks failing